Makefile

```diff
@@ -99,8 +102,14 @@ endif
 install:
 	gprinstall -f -P gnat/lsp_server.gpr -p -r --mode=usage \
 		--prefix=$(DESTDIR) $(LIBRARY_FLAGS)
-	gprinstall -f -P gnat/tester.gpr -p --prefix=$(DESTDIR) $(LIBRARY_FLAGS)
-	gprinstall -f -P gnat/codec_test.gpr -p --prefix=$(DESTDIR) $(LIBRARY_FLAGS)
+	gprinstall -f -P gnat/lsp_client.gpr -p -r	\
```

Comment:
The install of lsp_client_glib.gpr should also be left out of this section.

Reply:
The first commit did install lsp_client_glib.gpr, this one doesn’t - or have I messed something?
The clean: target should clean up lsp_client.gpr (and lsp_client_glib.gpr).
Should there be a non-default, temporary, option to build & install lsp_client_glib.gpr?

Reply:
My bad! I was reviewing the first commit indeed.

Reply:
I think we're okay without specific handling of the *glib bits.